src/gui/transferlistfilterswidget.cpp

```diff
@@ -521,6 +522,53 @@ void TrackerFiltersList::removeItem(const QString &trackerURL, const BitTorrent:
         applyFilter(currentRow());
 }
 
+void TrackerFiltersList::resetItems(const QStringList &trackers, const BitTorrent::TorrentID &id)
+{
+    m_errors.remove(id);
+    m_warnings.remove(id);
+
+    for (auto iter = m_trackers.begin(); iter != m_trackers.end();)
```

Comment:
It looks like you can utilize `Algorithm::removeIf()` for removing `m_trackers` entires and simplify some logic here.

Reply:
> It looks like you can utilize `Algorithm::removeIf()` for removing `m_trackers` entires and simplify some logic here.

Done.